### DISCUSSION TITLE
Pay as go on vercel platform

### DISCUSSION THREAD

**Discussion by ats1999** [vercel/vercel#5522] [Help]
Vercel provides pay-as-you-go features. When I visit its pricing section, I got it has minimum price is *20$/mo* per member. 

I am a single developer of my website. I don't know when my site will gain popularity. Currently I have 10 visitor per day on my site. 

So if I have to start a project with this platform then, do I have to pay $20 per month. Even my site traffic is very low. 

What is pay as you go?

**Comment by TooTallNate** [comment]
You should start out with the Hobby plan, which is free. See the [Pricing](https://vercel.com/pricing) page for more details.

**Reply by ats1999** [reply]
> You should start out with the Hobby plan, which is free. See the [Pricing](https://vercel.com/pricing) page for more details.

In hobby plan i can't put ads on my website. This is a restriction,in hobby plan that you can't do anything for money.

**Comment by theprasant** [comment]
@TooTallNate can I put ads if I add costume domain to site hosted on vercel with paying (or any suscription) ? @ats1999
